src/app/workflow/executions/executions-tab.component.html

```diff
@@ -0,0 +1,111 @@
+<div *ngIf="!ifMetricsExist">
+  <mat-card class="alert alert-warning mat-elevation-z" role="alert" data-cy="no-metrics-banner">
+    <mat-icon class="alert-warning-icon">info</mat-icon> No metrics data exist for this version.
+  </mat-card>
+</div>
+
+<div *ngIf="ifMetricsExist">
+  <div class="metrics-container">
+    <div class="w-25">
+      <div class="partner-dropdown">
+        <span class="metrics-header">Partner: </span>
+        <mat-form-field data-cy="metrics-partner-dropdown">
+          <mat-select [value]="currentPartner" (selectionChange)="matSelectChange($event)">
```

Comment:
It would be nice to have a pipe for this; in your screenshot, which is great, it shows `DNA_STACK`, which is our enum, not how the display their name (`DNAstack`). Looks like there is some code (not a pipe, alas) that already does [some of it](https://github.com/dockstore/dockstore-ui2/blob/1cd20066f733c1997ba77675bb59a0bbc2f9ef55/src/app/workflow/launch-third-party/multi-cloud-launch/multi-cloud-launch.component.ts#L81). 

If this is going to be in preview mode (see my other comment/question), can wait for 1.15 with a followup ticket.

Reply:
But agreed, it looks unfinished with the enum value. We probably also need to do something with a display value for the validators (MINIWDL)

Reply:
Ticket created: https://ucsc-cgl.atlassian.net/browse/SEAB-5413